smoke/_init.sh

```
@@ -332,20 +332,43 @@ fi
+	local count=1
+	local timeout=5
+	while getopts "c:t:" opt; do
+		case $opt in
+		c) count="$OPTARG" ;;
+		t) timeout="$OPTARG" ;;
+		*) echo "error: invalid arguments" >&2; return 1 ;;
+		esac
+	done
+	shift $((OPTIND - 1))
+	local pattern="$1"
+	{ tail -f -n +$((__event_mark + 1)) $tmp/events.log || : ; } | \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)